cms/standard_pages/utils.py

```diff
@@ -0,0 +1,12 @@
+from django.conf import settings
+
+from cms.standard_pages.models import CookiesPage
+
+
+def get_cookies_page_url(language_code: str) -> str | None:
+    if language_code not in {supported_lang[0] for supported_lang in settings.LANGUAGES}:
+        # Fallback to the default english language code if the provided language code is not supported
+        language_code = settings.LANGUAGE_CODE
+    # Cookies pages must exist for supported languages
+    cookies_page = CookiesPage.objects.get(locale__language_code=language_code)
+    return cookies_page.get_url()
```

Comment:
Might be worth handling `CookiesPage.DoesNotExist` for stability, even if the migration does create this page, and your hooks prevent things from being deleted; only because this would be done for every request, and as such the whole site would crash in that instance.